dbt/include/bigquery/macros/materializations/incremental.sql

```diff
@@ -46,6 +46,12 @@
         )
       {%- endset -%}
 
+      {#-- Because we're putting the model SQL _directly_ into the MERGE statement,
```

Comment:
do we need comment symbols on each line?

Reply:
jinja does support comment blocks if thats what your referring to https://tedboy.github.io/jinja2/templ5.html, https://github.com/HubSpot/jinjava/blob/master/src/test/resources/parse/tokenizer/multiline-comment.jinja.

Reply:
The reason to put in the SQL comment lines is not because they help comment but because it makes the file highlighting look better :)

Reply:
An open question regarding that `sql_header`: is there a plan to detect them automatically these kind of headers to avoid the "unfriendly" approach of wrapping the code in a `{%- call set_sql_header(config) %}` block.
It requires dedicated documentation and mentoring on our end and I feel it could be simplified but maybe it's too much work for the overall usage?